frontends/election-manager/src/screens/manual_data_import_precinct_screen.test.tsx

```diff
@@ -335,68 +444,17 @@ test('can enter data for yes no contests as expected', async () => {
   );
 });
 
-test('loads prexisting manual data to edit', () => {
-  const talliesByPrecinct = getEmptyExternalTalliesByPrecinct(
-    electionSampleDefinition.election
```

Comment:
This test broke because it wasn't using properly typed data, which now matters. In order to make it less cumbersome, I rewrote it to be simpler, use `buildExternalTally`, and not check all values / precincts.